test/test_Data.py

```diff
@@ -22,7 +23,8 @@ def remove_entities(self, utils, classes):
 
     def test_Loader(self, utils, insert_data_from_csv):
         classes = ["_Image", "_Descriptor", "Person", "_BoundingBox"]
-        self.remove_entities(utils, classes)
+        self.remove_entities(utils, classes[2:3])
```

Comment:
remove_all_objects() below should also remove entities of class "Person", right?